description: Modified version of prefs.patch from Debian's Chromium that points to the correct location

--- a/chrome/browser/first_run/first_run_internal_linux.cc
+++ b/chrome/browser/first_run/first_run_internal_linux.cc
@@ -19,9 +19,7 @@ bool IsOrganicFirstRun() {
 
 base::FilePath MasterPrefsPath() {
   // The standard location of the master prefs is next to the chrome binary.
-  base::FilePath master_prefs;
-  if (!PathService::Get(base::DIR_EXE, &master_prefs))
-    return base::FilePath();
+  base::FilePath master_prefs = base::FilePath("/etc/chromium.d");
   return master_prefs.AppendASCII(installer::kDefaultMasterPrefs);
 }
 

